src/Microsoft.Diagnostics.Monitoring.WebApi/Utilities/StackUtilities.cs

```diff
@@ -19,6 +20,66 @@ internal enum StackFormat
 
     internal static class StackUtilities
     {
+        public static Models.CallStackResult TranslateCallStackResultToModel(CallStackResult result)
```

Comment:
Change this to be `Models.CallStack TranslateCallStackToModel(CallStack stack)` and update `JsonStacksFormatter.FormatStack` to use this method too.